index.ts

```diff
@@ -39,6 +39,7 @@ function adapter<StateT, CustomT, RequestContext>(
       });
       ctx.status = result.status;
       ctx.body = result.value.value;
+      ctx.set(result.headers);
```

Comment:
so this merges headers with whatever some middleware has set (according to https://github.com/koajs/koa/blob/master/docs/api/response.md#responsesetfields)